web/sites/default/config/language/es/webform.webform_options.ethnicity.yml

```diff
@@ -1,10 +1,15 @@
 label: 'Race and Ethnicity'
 category: Demographic
 options: |-
-  'American Indian or Alaska Native': 'Indio americano, Indígena de Estados Unidos o Nativo de Alaska'
-  Asian: Asiático
-  'Black or African American': 'Negro o Afroamericano'
-  'Latino/a/e/x or Hispanic': 'Latino o hispano'
-  'Middle Eastern/North African': 'Medio Oriente/África del Norte'
-  'Native Hawaiian or Pacific Islander': 'Nativo de Hawái o isleño del Pacífico'
-  White: Blanco
+  'American Indian/Alaska Native': 'American Indian/Alaska Native'
```

Comment:
I have to figure out what's going on here... my local environment does this sometimes too. For some reason it's trying to revert the Spanish translation back to English. Can we remove this change for now?